pyproject.toml

```diff
@@ -193,7 +195,7 @@ version = ">=1.4.1,<1.7.3"
 python = "~=3.7.0"
 
 [[tool.poetry.dependencies.scipy]]
-version = ">=1.4.1,<1.9.0"
+version = ">=1.4.1,<1.8.1"
```

Comment:
The fixed version seems to be 1.10.0 https://storage.googleapis.com/rasa-container-scan-reports/rasa-plus-dependency-report-3.6.6-20230828.txt

Reply:
I wasn't sure if I should reduce the range from a wider  `>=1.4.1,<1.9.0` to just `>=1.10.0`?
Is that better than reducing the range to versions before the vulnerability? `version = ">=1.4.1,<1.8.1"`

Reply:
hmm not sure, @rasa-jmac do you have a suggestion here?

Reply:
It's always better to move to at least the fixed version, which is indeed `>=1.10.0`

I don't see any docs that say the vulnerability _wasn't_ present in <1.8.1, just that that is the vulnerable version we have installed. Indeed, the [vuln listing](https://nvd.nist.gov/vuln/detail/CVE-2023-25399) (see "Known Affected Software Configurations") suggests it is every release before 1.10.0.

Also, even if docs say a version isn't present in an older version, we don't really want to be shipping older versions of software as a fix for things. We wouldn't be able to say for sure if the vuln really isn't present, or if the maintainers just didn't check back that far to see if it was.

Reply:
That makes sense. Thanks for your input!

Reply:
Also, by moving backwards to older versions, we're more likely to reintroduce older vulns that have already previously been fixed, and actually causing a larger problem than the one we're trying to fix.